tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -276,6 +279,24 @@ void ral_gemm(ExecutionContext* ctx, void* stream_handle, MemRefType<InT, 2> A,
     return;
   }
 
+#if defined(PLATFORM_ALIBABA) and defined(ENABLE_BLADE_GEMM)
```

Comment:
if we are going to move cutlass into BladeDISC, is it better to just name it as "cutlass_gemm" other than "blade_gemm"? A more straightforward name with less 'concept' should be more friendly to the users. Don't know what other people think.

Reply:
The blade_gemm() is a general interface that may call different implementations with jit tuning. Simply calling it cutlass_gemm is miss-leading.

Reply:
`blade_gemm` seems better is we're going to introduce more cutlass-like kernels sources. BTW, isn't pretty much like patine client ?

Reply:
I'm curious about some details about `blade_gemm`, better to have some design doc to introduce the general interface `blade_gemm` and the backend of it. 
And another, Does the exciting MKL gemm and CDUA gemm is a sub-module of `blade_gemm`?

Reply:
> I'm curious about some details about `blade_gemm`, better to have some design doc to introduce the general interface `blade_gemm` and the backend of it. And another, Does the exciting MKL gemm and CDUA gemm is a sub-module of `blade_gemm`?

So far blade_gemm() is an internal function, so it doesn't make sense to expose details about it in this place. Descriptions about the interface and implementions will be added in the definition of  blade_gemm() when it gets open sourced.